Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -95,6 +92,10 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 				ClearSizeEstimate();
 				UpdateVerticalScrollBarVisibility();
 				UpdateHorizontalScrollBarVisibility();
+
+				// This should always be the last call here so the initial selection does not get cleared
```

Comment:
It looks like `UpdateSelectionMode` is _supposed_ to clear the initial selection. I'm not really seeing any other places where the selection gets cleared. Where was it getting cleared? Do we have a bug with the `SelectionMode` instead?